nncf/experimental/onnx/datasets/imagenet_dataset.py

```diff
@@ -11,7 +11,7 @@
  limitations under the License.
 """
 
-from typing import Optional, Tuple
```

Comment:
In NNCF styleguide https://github.com/openvinotoolkit/nncf/blob/develop/docs/styleguide/PyGuide.md we have the rule - Imports should usually be on separate lines

Reply:
Yes, but typing module looks exceptional.
![image](https://user-images.githubusercontent.com/97221135/183320664-93bdbc8a-82f3-44b0-b372-e63e4bf91d6c.png)